crates/string-offsets/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "string-offsets"
+authors = ["The blackbird team <support@github.com>"]
+version = "0.1.0"
+edition = "2021"
+description = "Offset calculator to convert between byte, char, and line offsets in a string."
```

Comment:
```suggestion
description = "Offset calculator to convert between byte (utf8 encoding), word (utf16 encoding), unicode characters, and line offsets in a string."
```

Reply:
I deleted some other stuff and ended up with:

> Converts string offsets between UTF-8 bytes, UTF-16 code units, Unicode code points, and lines.